_providers/daleth.cafe.md

```diff
@@ -0,0 +1,26 @@
+---
+name: Csl. Thomas Lüchow
+status: Teacher and Coach for Adults
+domains: 
+  - daleth.cafe
+server:
+  - type: IMAP
+    socket: TLS
+    hostname: daleth.cafe
+    port: 993
+    username_pattern: EMAIL
+  - type: SMTP
+    socket: STARTTLS
+    hostname: daleth.cafe
+    port: 587
+    username_pattern: EMAIL
+before_login_hint: Wellcome to mehl.cloud
+after_login_hint: Now you are loged in
```

Comment:
```suggestion
```

I would leave out those two lines, they are only meant for giving users tips on how to configure the server. With chatmail, this is not needed.